### ISSUE TITLE
Mysql variables common name issues

### ISSUE THREAD

**Issue by AurelDragut** [Laravel-Backpack/CRUD#261]
I have made a table named "case" (the romanian for "houses") and i got an error because of the term "case" which seems to be a mysql keyword or something. So i might sugest a little change in the CrudTrait.php file:

```
$type = DB::select(DB::raw('SHOW COLUMNS FROM `'.Config::get('database.connections.'.env('DB_CONNECTION').'.prefix').$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
``` 

(i added some `` around the table name)

**Comment by OwenMelbz** [comment]
Hi @AurelDragut 

Im making a consolidated list https://github.com/Laravel-Backpack/CRUD/issues/285 so will be closing this for the time being to keep everything tidy.

Thats not to say this will be forgotten, however if you can create a PR for it, I'm sure @tabacitu will be able to review it at some point

**Comment by tabacitu** [comment]
Fixed, thank you! It will be available in the next point release (3.1.49 - today or tomorrow).

Cheers!
